authorization/README.md

```diff
@@ -0,0 +1,35 @@
+# Tapsi Pack API: Authorization Guide
```

Comment:
There are some misinformation and potential misunderstanding in this document. the definition of API secret key and OAuth method is not clear and the differences too.

I guess it would be nice if we could have these sections:
1. Definitions
2. Usages
3. Differences (How to choose section) (what is the best for you section)

Reply:
fixed